### PR TITLE
feat(query): added "SQL DB Instance With SSL Disabled" for Google Deployment Manager

### PR DESCRIPTION
**Pull request by joaorufi**
Signed-off-by: joaorufi <joao.rufino@checkmarx.com>

**Proposed Changes**
- Added "SQL DB Instance With SSL Disabled" for Google Deployment Manager

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/googleDeploymentManager/sql_db_instance_with_ssl_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+	not common_lib.valid_key(settings.ipConfiguration, "requireSsl")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("resources.name={{%s}}.properties.settings.ipConfiguration.requireSsl", [resource.name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "'settings.ipConfiguration.requireSsl' is defined and not null",
+		"keyActualValue": "'settings.ipConfiguration.requireSsl' is undefined or null", 
+		"searchLine": common_lib.build_search_line(["resources", idx, "properties", "settings", "ipConfiguration", "requireSsl"], []),
```
Comment (rafaela-soares) [inline assets/queries/googleDeploymentManager/sql_db_instance_with_ssl_disabled/query.rego:35]:
```suggestion
		"searchLine": common_lib.build_search_line(["resources", idx, "properties", "settings", "ipConfiguration"], []),
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
LGTM 🚀I have only a few suggestions:

File: assets/queries/ansible/gcp/sql_db_instance_with_ssl_disabled/metadata.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "queryName": "SQL DB Instance With SSL Disabled",
   "severity": "HIGH",
   "category": "Encryption",
-  "descriptionText": "Cloud SQL Database Instance with SSL disabled for incoming connections",
+  "descriptionText": "Cloud SQL Database Instance should have SLL enabled",
```
Comment (rafaela-soares) [inline assets/queries/ansible/gcp/sql_db_instance_with_ssl_disabled/metadata.json:6]: Can you also update it in `kics/assets/queries/terraform/gcp/sql_db_instance_with_ssl_disabled/metadata.json`, please?

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/ansible/gcp/sql_db_instance_with_ssl_disabled/metadata.json (hunk 2 above)
Comment (JoaoDanielRufino) [reply assets/queries/ansible/gcp/sql_db_instance_with_ssl_disabled/metadata.json:6]: Done!

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Pull request merged**